docs/core/diagnostics/debug-linux-dumps.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Debugging Linux Dumps
+description: In this article, you'll learn how to collect and analyze dumps from Linux environments.
+ms.date: 08/27/2020
+---
+
+# Debugging Linux Dumps
+
+**This article applies to: ✔️** .NET Core 3.0 SDK and later versions
+
+## Collecting dumps on Linux
+
+The two recommended ways of collecting dumps on Linux are the [`dotnet-dump`](dotnet-dump.md) or [`createdump`](https://github.com/dotnet/runtime/blob/master/docs/design/coreclr/botr/xplat-minidump-generation.md) tools.
+
+### Managed dumps with `dotnet-dump`
+
+The [`dotnet-dump`](dotnet-dump.md) tool is the simplest to use and does not have a dependency on any native debuggers, so it works on a wide variety of Linux platforms (like Alpine or ARM32/ARM64) where traditional debugging tools may not be available. Note, however, that `dotnet-dump` only captures managed state so it can't be used for debugging issues in native code. Dumps collected by `dotnet-dump` must be analyzed in an environment with the the same OS and architecture the dump was created on. The [`dotnet-gcdump`](dotnet-gcdump.md) tool can be used as an alternative that only captures GC heap information but produces dumps that can be analyzed on Windows.
```

Comment:
```suggestion
The [`dotnet-dump`](dotnet-dump.md) tool is simple to use, and does not have a dependency on any native debuggers. `dotnet-dump` works on a wide variety of Linux platforms (like Alpine or ARM32/ARM64) where traditional debugging tools may not be available. However, `dotnet-dump` only captures managed state so it can't be used for debugging issues in native code. Dumps collected by `dotnet-dump` are analyzed in an environment with the the same OS and architecture the dump was created on. The [`dotnet-gcdump`](dotnet-gcdump.md) tool can be used as an alternative that only captures GC heap information but produces dumps that can be analyzed on Windows.
```